Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:lts-alpine
+
+VOLUME [ "/data" ]
+ENTRYPOINT [ "/home/excalidraw/bin/run" ]
+
+RUN addgroup -S excalidraw && adduser -S excalidraw -G excalidraw
+RUN apk add --no-cache \
+    python \
+    g++ \
+    build-base \
+    cairo-dev \
+    jpeg-dev \
+    pango-dev \
+    musl-dev \
+    giflib-dev \
+    pixman-dev \
+    pangomm-dev \
+    libjpeg-turbo-dev \
+    freetype-dev
+
+USER excalidraw
+WORKDIR /home/excalidraw
+
+COPY --chown=excalidraw:excalidraw package.json .
+RUN yarn -s
+RUN yarn global add typescript
```

Comment:
Not quite sure what the following line is doing here (I don't see any `tsc` call elsewhere in the Dockerfile, plus it's already a `devDependency`), otherwise the PR sounds good.

Reply:
Woops I also missed the notification this time. To many repos to many notifications. If I recall accordingly there was an error/warning without typescripts last time I tried, will rebase the change on top of current master and try again to figure out.

Reply:
I have removed it. Can't figure the need for it anymore.
Without it also works.